db/models/employee.js

```diff
@@ -0,0 +1,32 @@
+const mongoose = require('mongoose');
+
+const employeeSchema = new mongoose.Schema({
+  firstName: {
+    type: String,
+    required: true,
+  },
+  lastName: {
+    type: String,
+    required: true,
+  },
+  profileImage: {
+    type: String,
+    required: false, 
+  },
+  department: {
+    type: String,
+    enum: ['Management', 'Administration', 'Design', 'Development'],
```

Comment:
can we specify this enum variables in some utils file for example utils/enums where will be all enums